src/StripeTests/Entities/StripeDiscountTest.cs

```diff
@@ -0,0 +1,19 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeDiscountTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            // TODO: Figure out how to test this resource.
```

Comment:
Flagging this one as one that needs a local fixture. There are a few of those.

Something tells me we should add fixture logic to stripe-mock for resources that can't be fetched instead of having local JSON files for each test suite where they go out of sync other time.

Reply:
I'm not sure that we want stripe-mock to be able to return any arbitrary resource without a "real" endpoint, but maybe we could embed the OpenAPI fixtures file in the library's test project. That would make it much easier to maintain and update.

Reply:
So it's not really "arbitrary resource". It's also "resources that are only returned on creation" for example like usage record. It forces us to have local files that go out of sync quickly (or are manually written incorrectly).